script/precompile-assets

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
it's not the cross-platform able solution, what about the switch to Node for cross-platform running?

Reply:
That's a good idea. I'll do that today.